build.gradle

```diff
@@ -20,9 +20,9 @@ ext {
     automatticRestVersion = '1.0.8'
     automatticStoriesVersion = '2.0.0'
     automatticTracksVersion = '2.2.0'
-    gutenbergMobileVersion = 'v1.92.0-alpha1'
+    gutenbergMobileVersion = '5596-5a7826bee6a11d22a2a96703009afb1b2c47fba2'
     wordPressAztecVersion = 'v1.6.3'
-    wordPressFluxCVersion = 'trunk-fcede535112327abf59416b7b544cf526c44fd00'
+    wordPressFluxCVersion = '2691-a3bac6077768944a3e79a6a1eb4c7c22dd2e1183'
```

Comment:
This references will be replaced once https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2691 and https://github.com/wordpress-mobile/gutenberg-mobile/pull/5596 are merged.